src/components/LoginLink.js

```diff
@@ -2,10 +2,18 @@ import React from 'react';
 
 import {config} from '../config/index';
 
-export function LoginLink({children, orgCode, ...props}) {
+export function LoginLink({children, postLoginRedirectURL, orgCode, ...props}) {
+  const params = new URLSearchParams(
+    JSON.parse(
+      JSON.stringify({
+        org_code: orgCode,
+        post_login_redirect_url: postLoginRedirectURL
+      })
```

Comment:
@peterphanouvong pretty sure you can ditch both JSON.parse and JSON.stringify here and just pass the object directly
```const params = new URLSearchParams({
        org_code: orgCode,
        post_login_redirect_url: postLoginRedirectURL
 ```